app_datetime/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+
+class DateTime(models.Model):
+  datetime = models.DateTimeField(_("Date Time"))
+  isUTC = models.BooleanField(_("Is UTC Time"), default=False)
```

Comment:
This is not required as we should be able to get `tzinfo` from `DateTimeField`

https://docs.python.org/3/library/datetime.html#datetime.tzinfo